setup.py

```diff
@@ -10,14 +10,15 @@
 setup(
     name='ctapipe_io_nectarcam',
     packages=find_packages(),
-    version='0.1',
+    version='0.1', #TODO implement versioning
     description='ctapipe plugin for reading NectarCam files',
     long_description=long_description,
     long_description_content_type='text/markdown',
     install_requires=[
-        'astropy',
-        'ctapipe',
-        'protozfits @ https://github.com/cta-sst-1m/protozfitsreader/archive/v1.5.0.tar.gz',
+        'astropy~=4.2',
+        'ctapipe~=0.12',
+        'protozfits~=2.0',
+        'setuptools_scm',
```

Comment:
This is not (yet) needed, see above

Reply:
Right, I had put it there to test because I had seen it is the approach for versioning seen in ctapipe. Shall I remove this for the time being?

Reply:
I opened PR #19